float32 vx_set
float32 vy_set
float32 wz_set
float32 yaw_add
float32 pitch_add
uint8 mode